test/ng/parseSpec.js

```diff
@@ -3232,6 +3232,18 @@ describe('parser', function() {
           expect(fn()).toEqual(undefined);
         }));
 
+        it('should invoke a stateless filter once when the parsed expression has an interceptor',
+           inject(function($parse, $rootScope) {
+          var countFilter = jasmine.createSpy();
+          var interceptor = jasmine.createSpy();
+          countFilter.and.callThrough();
```

Comment:
I do not understand this. `countFilter` is registered as a filter and then this filter is used in the expression `foo() | count`. `foo` is a function that returns `1` and i want to be sure that the expression evaluates to something other than `undefined`